### DISCUSSION TITLE
SSL: CERTIFICATE_VERIFY_FAILED

### DISCUSSION THREAD

**Discussion by rkoverov** [deepgram/community#718] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram SDKs

### Details

Hi, I also didn't find any solution to this problem.
I am encountering an error while attempting to start the streaming example.
WebSocketException in LiveClient.start: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007).

Could you please help me troubleshoot the issue?

The pre-recorded examples worked perfectly.
OS: Ubuntu v. 22.04.4 LTS.

Thanks in advance.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
You will need to do updates to your OS cert authorities. This is a common problem on macos, but also seems to be now happening on ubuntu.

**Comment by rkoverov** [comment]
I updated the certificate authorities on my Ubuntu system, but the issue persisted.

**Comment by rkoverov** [comment]
All is OK while using JavaScript example.

**Comment by davidvonthenen** [comment, marked as answer]
You need to do this:
https://stackoverflow.com/questions/42982143/python-requests-how-to-use-system-ca-certificates-debian-ubuntu

**Comment by rkoverov** [comment]
David,
Thank you. It works!
